x/wasm/ioutils/ioutil.go

```diff
@@ -3,17 +3,16 @@ package ioutils
 import (
 	"bytes"
 	"compress/gzip"
+	"fmt"
 	"io"
-
-	errorsmod "cosmossdk.io/errors"
-
-	"github.com/CosmWasm/wasmd/x/wasm/types"
 )
 
+var errLimit = fmt.Errorf("exceeds limit")
```

Comment:
Nit: as there is no params to fmt you can use:
```suggestion
var errLimit = errors.New("exceeds limit")
```